enclave/core/string.c

```diff
@@ -89,6 +89,21 @@ size_t oe_strlcpy(char* dest, const char* src, size_t size)
     return (size_t)(src - start);
 }
 
+char* oe_strncpy(char* dest, const char* src, size_t n)
+{
+    char* ret = dest;
```

Comment:
While this does not explicitly impact any of our existing scenario, should we following the way in MUSL's strncpy implementation, which follows the C standard by null-padding the destination bytes?

Reply:
Thank you Cheng-mean. I removed this function since it is not used anywhere.